### PR TITLE
535: Log failed activation attempts

### PR DESCRIPTION
**Pull request by sarahsporck**

### CODE REVIEW

File: backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardMutationService.kt [hunk 1]
```diff
@@ -26,48 +27,50 @@ class CardMutationService {
...
+            context.backendConfiguration.projects.find { it.id == project }
+                ?: throw NullPointerException("Project not found")
         val cardHash = Base64.getDecoder().decode(cardInfoHashBase64)
         val activationSecret = Base64.getDecoder().decode(activationSecretBase64)
         val card = transaction {
             CardRepository.findByHashAndActivationSecret(project, cardHash, activationSecret)
         }
 
         if (card == null) {
+            logger.info("${context.remoteIp} failed to activate")
```
Comment (michael-markl) [inline backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardMutationService.kt:73]:
```suggestion
            logger.info("${context.remoteIp} failed to activate entitlement card.")
```

**Review by michael-markl — APPROVED** [review]
I think we need to talk to @svenseeberg so he can adjust the fail2ban filter?

**Comment by maxammann** [comment]
> I think we need to talk to @svenseeberg so he can adjust the fail2ban filter?

Yes, I assigned myself. Maybe I can have a look today.

File: backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardMutationService.kt (hunk 1 above)
Comment (maxammann) [inline backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardMutationService.kt:73]:
```suggestion
            logger.info("${context.remoteIp} failed to activate entitlement card")
```

**Review by maxammann — APPROVED** [review]

File: backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardMutationService.kt (hunk 1 above)
Comment (maxammann) [reply backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardMutationService.kt:73]: no dot else regex very very complicated

**Pull request merged**